modules/react-components/src/icon/generic-icon.tsx

```diff
@@ -117,6 +117,7 @@ export const GenericIcon: React.FunctionComponent<PropsWithChildren<GenericIconP
 
             // Check if the icon is a module and has `ReactComponent` property.
             // Important when used with SVG's imported with `@svgr/webpack`.
+            // eslint-disable-next-line no-prototype-builtins
             if (icon.hasOwnProperty("ReactComponent") && typeof icon.ReactComponent === "function") {
```

Comment:
This can be replaced with Object.prototype.hasOwnProperty.call(icon, "ReactComponent"). Then we don't need to disable the lint warning.